Number of train_response logs 58926
[Epoch 1]
Training: 100%|███████████████████████████████████████████████████| 231/231 [00:17<00:00, 13.25it/s]
Average loss: 0.6856778612384549
Evaluating: 100%|███████████████████████████████████████████████████| 33/33 [00:01<00:00, 30.62it/s]
Traceback (most recent call last):
  File "d:\Git\IDCDF-WKY\run.py", line 65, in <module>
    sys.exit(main(config_dict))
             ^^^^^^^^^^^^^^^^^
  File "d:\Git\IDCDF-WKY\run.py", line 62, in main
    idcdm.train(datahub, "train", "valid", valid_metrics=validate_metrics, batch_size=config['batch_size'],epoch=config['epoch'], lr=config['lr'])
  File "d:\Git\IDCDF-WKY\inscd\models\idcdm.py", line 45, in train
    self._train(datahub=datahub, set_type=set_type,
  File "d:\Git\IDCDF-WKY\inscd\_base.py", line 52, in _train
    self.score(datahub, valid_set_type, valid_metrics, **kwargs)
  File "d:\Git\IDCDF-WKY\inscd\models\idcdm.py", line 57, in score
    return self._score(datahub=datahub, set_type=set_type, metrics=metrics, batch_size=batch_size)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\Git\IDCDF-WKY\inscd\_listener.py", line 33, in wrapper
    result = self.__format(func(*args, **kwargs))
                           ^^^^^^^^^^^^^^^^^^^^^
  File "d:\Git\IDCDF-WKY\inscd\_base.py", line 67, in _score
    return ruler(self, datahub, set_type, pred_r, metrics)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\Git\IDCDF-WKY\inscd\_ruler.py", line 158, in __call__
    results[metric] = self.__method_map[metric](true_r, pred_r)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\Git\IDCDF-WKY\inscd\_ruler.py", line 32, in accuracy
    return accuracy_score(true_r, pred_r)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin.DESKTOP-7USASVH\AppData\Local\Programs\Python\Python312\Lib\site-packages\sklearn\utils\_param_validation.py", line 213, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin.DESKTOP-7USASVH\AppData\Local\Programs\Python\Python312\Lib\site-packages\sklearn\metrics\_classification.py", line 231, in accuracy_score
    y_type, y_true, y_pred = _check_targets(y_true, y_pred)
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin.DESKTOP-7USASVH\AppData\Local\Programs\Python\Python312\Lib\site-packages\sklearn\metrics\_classification.py", line 112, in _check_targets
    raise ValueError(
ValueError: Classification metrics can't handle a mix of binary and continuous targets
